pkg/kritis/admission/admission.go

```diff
@@ -178,11 +178,17 @@ func ReviewHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func reviewDeployment(deployment *appsv1.Deployment, ar *v1beta1.AdmissionReview) {
+	images := DeploymentImages(*deployment)
+	// check if the Deployments's owner has already been validated
+	if checkOwners(images, &deployment.ObjectMeta) {
+		return
```

Comment:
Can we log here and elsewhere so its easier to know why the object was admitted.